course_catalog/serializers.py

```diff
@@ -399,6 +399,7 @@ class ProgramSerializer(serializers.ModelSerializer, FavoriteSerializerMixin):
 
     items = ProgramItemSerializer(many=True, allow_null=True)
     topics = CourseTopicSerializer(read_only=True, many=True, allow_null=True)
+    prices = CoursePriceSerializer(read_only=True, many=True, allow_null=True)
```

Comment:
Do we need to expose all of the prices via REST API? If we only need the most recent one we should expose that specifically.

Reply:
Program prices don't have any dates (besides created/modified), and for xpro at least there should be only one anyway.  I don't think the micromasters catalog API returns any prices at all yet.